.github/workflows/moving-cards.yml

```diff
@@ -92,16 +100,24 @@ jobs:
         uses: xom9ikk/dotenv@v1.0.2
         with:
           path: .github/workflows/
+      # NOTE: The assignment rules has to be changed when the new column for automated PR is added
```

Comment:
I'd create the column now, I'd add the ID to the .env file and will start leaving the "pending changes" commented to ensure in the future it's covered.

Reply:
I didn't do it because this PR is complex and I didn't want to add more new features